### ISSUE TITLE
TextBox doesn't work with Formik

### ISSUE THREAD

**Issue by jchn-codes** [telerik/kendo-react#1709]
Hi,

as title says: When using the TextBox component, Formik throws errors onChange:

```
Warning: Formik called `handleChange`, but you forgot to pass an `id` or `name` attribute to your input: undefined
    Formik cannot determine which value to update. For more info see https://formik.org/docs/api/formik#handlechange-e-reactchangeeventany--void
```

Same as https://github.com/telerik/kendo-react/issues/799

**Comment by nstoychev** [comment]
Fixed in our dev channel version 6.0.3-dev.202310161425.

**Comment by nstoychev** [comment]
We published [6.1.0](https://www.telerik.com/kendo-react-ui/components/changelogs/ui-for-react/#v6.1.0).
